Capabilitystatements/capabilitystatement-client-gp-patientdata.xml

```diff
@@ -118,6 +118,10 @@
                 <name value="code"/>
                 <type value="token"/>
             </searchParam>
+            <searchParam>
+                <name value="date"/>
+                <type value="date"/>
+            </searchParam>
         </resource>
```

Comment:
Ik weet niet of we de scope van het ticket te veel oprekken, maar moeten de _include's er niet ook bij?

Reply:
De _includes worden nu niet benoemd als SearchParameters in de andere CapabilityStatements, vind ik dus lastig of dat hierbij hoort.